src/AppContext.tsx

```diff
@@ -257,6 +287,20 @@ export const AppContextProvider: React.FC = ({children}) => {
     restartShareTravelHabitsOnboarding,
   } = useMemo(
     () => ({
+      completeExtendedOnboarding: async () => {
+        await storage.set(
+          storeKey.extendedOnboardingOnboarded,
+          JSON.stringify(true),
```

Comment:
It can be also just as 'true'

Reply:
Yep. I think AppContext could use a refactor to simplify and have less duplicate code for each variable.
For now keeping it like this since it follows the exact pattern of all the other ones